### ISSUE TITLE
LDNS installation fails with the following errors in Debian Jessie 

### ISSUE THREAD

**Issue by sandoche2k** [dotse/zonemaster-ldns#30]
```
src/ldns/dane.c: In function ‘ldns_dane_pkix_get_last_self_signed’:
src/ldns/dane.c:330:14: error: dereferencing pointer to incomplete type ‘X509_STORE_CTX {aka struct x509_store_ctx_st}’
  if (vrfy_ctx->error == X509_V_ERR_SELF_SIGNED_CERT_IN_CHAIN ||
              ^~
Makefile:728: recipe for target 'src/ldns/dane.o' failed
make: *** [src/ldns/dane.o] Error 1
  /home/sandoche/Zonemaster-LDNS-1.0.2/.
  /usr/bin/make -- NOT OK
```

**Comment by pawal** [comment]
Yes, I have mentioned this a couple of times... Current Zonemaster::LDNS does not support modern OpenSSL versions.

**Comment by matsduf** [comment]
This issue is related to issue #10.

We are aware of the problem, but we have not had enough resources to resolve it.

**Comment by matsduf** [comment]
This issue will probably be solved by PR #41.

**Comment by matsduf** [comment]
Solved by #41
